backend/s3/s3.go

```diff
@@ -2393,6 +2404,7 @@ type Options struct {
 	VersionAt             fs.Time              `config:"version_at"`
 	Decompress            bool                 `config:"decompress"`
 	MightGzip             fs.Tristate          `config:"might_gzip"`
+	ForceGzipHeader       bool                 `config:"force_gzip_header"`
```

Comment:
This should be an `fs.Tristate`.

The default is `auto` which should use the provider default. The user can set it explicitly.

----

This also needs to appear in the Options above which is where the help docs go.

Note that not having this header breaks downloading files with `Content-Encoding: gzip` so it might be worth experimenting with that before writing the help.

----

I'm not 100% happy with the name `ForceGzipHeader`, how about  calling it `UseAcceptEncodingGzip`?

Reply:
Fine by me! PR updated 👍🏻